Added report for three npm packages

6mile opened the pull request:
Added report for three malicious npm packages: lana-docs-site, grafana-sentry-datasource, and spectro-cloud-docs

6mile commented:
heya @calebbrown there was a historical advisory report for spectro-cloud-docs package (MAL-2023-8700.json).  I didn't know how you wanted to handle it so I just added a net new report and removed the historical one from my commit.  IF this is not how you want to handle overlaps like this, lemme know and I'll adjust.

calebbrown commented:
> heya @calebbrown there was a historical advisory report for spectro-cloud-docs package (MAL-2023-8700.json). I didn't know how you wanted to handle it so I just added a net new report and removed the historical one from my commit. IF this is not how you want to handle overlaps like this, lemme know and I'll adjust.

Hi @6mile, can you undelete the report MAL-2023-8700.json? The new file will be collapsed into the existing report. We have a policy of not deleting reports.

Thanks!

6mile commented:
> > heya @calebbrown there was a historical advisory report for spectro-cloud-docs package (MAL-2023-8700.json). I didn't know how you wanted to handle it so I just added a net new report and removed the historical one from my commit. IF this is not how you want to handle overlaps like this, lemme know and I'll adjust.
> 
> Hi @6mile, can you undelete the report MAL-2023-8700.json? The new file will be collapsed into the existing report. We have a policy of not deleting reports.
> 
> Thanks!

Heya @calebbrown I've added MAL-2023-8700.json back in.  Cheers
